spec/encrypt_attributes_spec.rb

```diff
@@ -70,4 +72,13 @@ def dynamic_secret_key
       expect(target.serialized_encoded).to eq value
     end
   end
+
+  context 'when :accessors option is specified in :serialize option' do
+    it 'creates accessors' do
+      value = 'あいうえお'
+      target.create_accessors_attribute = value
+      expect(target.create_accessors).to eq ({ attribute: value })
+      expect(target.create_accessors_attribute).to eq value
+    end
+  end
```

Comment:
Note that we are not dealing with the issue of removing key/value pairs, i.e. `target.create_accessors_attribute2 = nil` will leave behind the key so you have `{ attribute1: value, attribute2: nil }`. I think we just have to assume that anything that tricky should be handled directly on the hash rather than through accessors.